Cargo.toml

```diff
@@ -13,13 +13,14 @@ categories = ["web-programming::http-server"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-actix-web = { version = "4.0.0-beta.9", default-features = false }
+actix-web = { version = "4.0.0-beta.13", default-features = false }
+actix-http = "=3.0.0-beta.14" # temporary
```

Comment:
Can this be removed? I didn't see it used in the code...

Reply:
Actually I could not compile without this (even before my patch). This is because a new version of actix-http is pulled by actix-web `4.0.0-beta.14` and it break things. That is why I fixed the version of actix-http. I can have a look on how to fix the whole thing with the latest versions but I feel like this would be better in a follow up PR (I have projects that would benefit from the current PR and are not ready to be upgraded to actix-web `4.0.0-beta.14` yet).

But maybe I'm wrong. Can you build `main` from scratch right now? (I mean if you delete your `Cargo.lock`)

Reply:
No, main doesn't compile. I just wasn't aware of that.

I'll merge it then, thanks a lot!